src/strings/csv/en.csv

```diff
@@ -652,7 +652,7 @@ OUNCES,Ounces
 OUT_PLANTING,Out-planting
 OUTPLANT,Outplant
 OUTPLANTS_REQUIRE_READY_SEEDLINGS,"Outplants require batches with seedlings that are ""ready."""
-OUTSTANDING,Outstanding,In the sense of Pending or Unresolved
+OUTSTANDING,Outstanding,In the sense of Pending, Unresolved, or Remaining to be Done
```

Comment:
oh you can remove my suggestion, was meant as a replacement.. got that from the interwebz and was also a first hunch for me.. so ran it by you
this is a csv.. those extra commas have some semantics..
revert and check in is fine